src/main/java/org/triplea/client/launch/screens/staging/GameStartedCallback.java

```diff
@@ -0,0 +1,9 @@
+package org.triplea.client.launch.screens.staging;
+
+/**
+ * Marker interface for listeners of the 'game started' event. Notably used by network hosts to send the
+ * 'game start' event to network clients.
+ */
+public interface GameStartedCallback {
```

Comment:
I'd suggest annotating it as `@FunctionalInterface` and/or making it extend the Runnable interface

Reply:
`@FunctionalInterface` makes sense, hard to think of additional methods we would want on this interface.

Extending Runnable is a bit more controversial, as we could just use `Runnable` to begin with and skip having the interface. The purpose of this class and method is to give friendly/usable names, using `gameIsStarted()` is desired vs `run()`.

Reply:
Marked as a functional interface in: 048fe10